pages/services/couchbase/1.0.0-6.0.0/uninstall/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: layout.pug
+navigationTitle:  Uninstall
+title: Uninstall
+menuWeight: 55
+excerpt: Uninstalling DC/OS Couchbase Services
+featureMaturity:
+enterprise: false
+model: /services/couchbase/data.yml
+render: mustache
+---
+
+## DC/OS 1.10
+
+If you are using DC/OS 1.10 or later:
+
+Uninstall the service from the DC/OS CLI with the command `dcos package uninstall <package_name>`. For example, to uninstall the {{ model.techName }} instance named `couchbase-dev`, run:
+
+  ```bash
+  dcos package uninstall couchbase-dev
+  ```
+  
+### Uninstall procedure
+
+Uninstalling the service consists of the following steps. The scheduler is relaunched in Marathon with the environment variable SDK_UNINSTALL set to “true”. This puts the Scheduler in an uninstall mode.
+
+The scheduler performs the uninstall with the following actions:
+
+   1. All running tasks for the service are terminated so that Mesos will reoffer their resources.
+   1. As the task resources are offered by Mesos, they are unreserved by the scheduler.
+   1. Once all known resources have been unreserved, the scheduler’s persistent state in ZooKeeper is deleted.
+
+<p class="message--warning"><strong>WARNING: </strong> Any data stored in reserved disk resources will be irretrievably lost.</p> 
+
+
+The cluster automatically removes the scheduler task once it advertises the completion of the uninstall process.
+
+<p class="message--warning"><strong>WARNING: </strong> Any data stored in reserved disk resources will be irretrievably lost.</p>  Once the uninstall operation has begun, it cannot be cancelled. Doing so may leave the service in an uncertain, half-destroyed state.</p> 
+
+
+
+### Debugging an uninstall
+
+In the vast majority of cases, this uninstall process goes off without a hitch. However, in certain situations, there can be snags along the way. For example, perhaps a machine in the cluster has permanently gone away, and the service being uninstalled had some resources allocated on that machine. This can result in the uninstall becoming stuck, because Mesos will never offer those resources to the uninstalling scheduler. Thus, the uninstalling scheduler will not be able to successfully unreserve the resources it had reserved on that machine.
+
+This situation is indicated by looking at the deploy plan while the uninstall is proceeding. The deploy plan may be viewed using either of the following methods:
+
+- CLI: `dcos couchbase --name=couchbase plan show deploy` (after running `dcos package install --cli couchbase` if needed)
```

Comment:
This sequence is backwards. Rewrite so that the first action (running `dcos package install --cli couchbase` is first in the sentence, then the second action.

Reply:
Added dcos package install --cli couchbase as a first step.